src/App/ActivityList.purs

```diff
@@ -214,21 +163,38 @@ priorityToColor priority =
     Medium -> "#f5f588"
     Low -> "#469dd0"
 
-type Slots = ( addTodoDialog :: forall query . H.Slot query AddTodoDialog.Output Int )
+type Slots = ( addTodoDialog :: forall query. H.Slot query AddTodoDialog.Output Int
+             , tagModal :: forall query. H.Slot query TagModal.Output Int
+             )
 
 _addTodoDialog = Proxy :: Proxy "addTodoDialog"
+_tagModal = Proxy :: Proxy "tagModal"
```

Comment:
@tmciver  the key part of this is the proxy containing the component function name correct? Meaning the underscore for the variable name `_tagModal` is just a convention correct?

Reply:
I don't completely understand this either.  But yes, I believe it is just a convention to use an underscore.  But I don't think it refers to the component's "function name" or the variable bound to the component because in this case the tag modal component is just called `component`.  I need to learn more about the use of `Proxy` by Halogen.

Reply:
Thanks, also need to dig in this. At this point it strikes me as magic which is a hint for me to start also looking deeper into the docs :).